lte/gateway/python/magma/subscriberdb/tests/rpc_tests.py

```diff
@@ -48,7 +49,7 @@ def setUp(self):
         port = self._rpc_server.add_insecure_port('0.0.0.0:0')
 
         # Add the servicer
-        self._servicer = SubscriberDBRpcServicer(store)
+        self._servicer = SubscriberDBRpcServicer(store, MagicMock())
```

Comment:
nit: update both params with keyword args to specify what `MagicMock` is being used for